catkin_ws/src/barracuda_vision/scripts/yolo_image_listener.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+
+import rospy
+from sensor_msgs.msg import Image
+from std_msgs.msg import Int8
+from darknet_ros_msgs.msg import BoundingBoxes, BoundingBox
+from inference_sdk import InferenceHTTPClient
+from cv_bridge import CvBridge
+import supervision as sv
+import torch
+from torchvision import transforms
+import os
+from ultralytics import YOLO
+import sys
+import base64
+
+# Initialize inference client for remote server
+client = InferenceHTTPClient(
+    api_url="http://localhost:9001",  # Use local inference server
```

Comment:
are these still being used? let's remove inference sdk stuff from this file if we are planning to use pytorch for inference. please go through this and find code that is not being used and remove it.